grunt/config/browserify.js

```diff
@@ -8,39 +8,21 @@ var UglifyJS = require('uglify-js');
 var uglifyify = require('uglifyify');
 var derequire = require('derequire');
 var aliasify = require('aliasify');
-var globalShim = require('browserify-global-shim');
 var collapser = require('bundle-collapser/plugin');
 
 var envifyDev = envify({NODE_ENV: process.env.NODE_ENV || 'development'});
 var envifyProd = envify({NODE_ENV: process.env.NODE_ENV || 'production'});
 
-var SECRET_INTERNALS_NAME = 'React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED';
-
-var shimSharedModulesFiles = {
-  './ReactCurrentOwner': SECRET_INTERNALS_NAME + '.ReactCurrentOwner',
-  './ReactComponentTreeHook': SECRET_INTERNALS_NAME + '.ReactComponentTreeHook',
-  // Use the global, anywhere we require React
-  './React': 'React',
-};
-
-// We can access these as absolute or relative. We need to shim both.
-for (var key in shimSharedModulesFiles) {
-  shimSharedModulesFiles[key.replace(/^\.\//, 'react/lib/')] = shimSharedModulesFiles[key];
-}
-
-var shimSharedModules = globalShim.configure(shimSharedModulesFiles);
-
-// Fiber needs the symbol from ReactElement right now. So we can't shim
-// ReactElement. Otherwise this would just be the same as above.
-// TODO: Refactor this so that Fiber can access the symbol without bringing in
-// the rest of ReactElement.
-var shimSharedModulesFiberFiles = {};
-for (var key in shimSharedModulesFiles) {
-  if (!/ReactElement/.test(key)) {
```

Comment:
I refactored this already so this special case for Fiber is no longer needed.

@zpao